app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -6,6 +6,14 @@ $govuk-cookie-banner-background: #d5e8f3;
   }
 }
 
+.gem-c-cookie-banner__hide-link {
```

Comment:
Is it possible this below the `.gem-c-cookie-banner` class so that the component CSS is at the top

Reply:
Thanks @DilwoarH - [done](https://github.com/alphagov/govuk_publishing_components/pull/843/commits/7fa4cc5d49bd9771bebd8bb9669c7386a1a660c3)!